bench/rust-analyzer-rs/optimized/45p2airz10aj25pm.ll

```diff
@@ -19033,61 +19033,62 @@ define internal fastcc noundef zeroext i1 @_ZN10hir_expand16builtin_fn_macro14us
   call void %11(ptr noalias nocapture noundef nonnull sret({ { i8, [23 x i8] }, i32, i32, { { i8, [15 x i8] }, i32, i8, i8, i8, [1 x i8] } }) align 8 dereferenceable(56) %3, ptr noundef nonnull align 1 %0, i32 noundef %8)
   %12 = getelementptr inbounds i8, ptr %3, i64 52
   %13 = load i8, ptr %12, align 4, !range !1860, !noundef !9
-  %14 = icmp ugt i8 %13, 1
+  %.05 = tail call i8 @llvm.ucmp.i8.i8(i8 %13, i8 2)
+  %switch = icmp ult i8 %.05, 2
```

Comment:
Duplicate of https://github.com/dtcxzyw/llvm-opt-benchmark/pull/1194#pullrequestreview-2238695484